jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/concurrent/atomic/AtomicIntegerFieldUpdater.java

```diff
@@ -79,11 +80,8 @@
     @CallerSensitive
     public static <U> AtomicIntegerFieldUpdater<U> newUpdater(Class<U> tclass,
                                                               String fieldName) {
-        return new AtomicIntegerFieldUpdaterImpl<U>(tclass, fieldName, null);
-        /* J2ObjC: Call stack not available.
         return new AtomicIntegerFieldUpdaterImpl<U>
-            (tclass, fieldName, Reflection.getCallerClass()); // android-changed
-        */
+            (tclass, fieldName, Reflection.getCallerClass());
```

Comment:
Nit: restore the "// android-changed" comment from the libcore source.

Reply:
The "// android-changed" comment is actually only in libcore-nougat but not libcore, so I delete it in this commit

Reply:
Perfect, thanks for the update.